--- app-3.20.1.57-orig.js	2019-12-01 12:49:27.326369400 -0500
+++ app.js	2019-12-01 12:49:43.065918800 -0500
@@ -5258,7 +5258,7 @@
                     M.debug("email not verified - verify if email belongs to locked domain");
                     var t = e.user.core.primaryEmail,
                         n = t.substring(t.indexOf("@") + 1);
-                    if (e.domains.list.indexOf(n) > -1) return !0
+                    if (e.domains.list.indexOf(n) > -1) return y.handleLoggedIn(e), !0
                 }
                 return !1
             }
@@ -9883,7 +9883,7 @@
             J = !1,
             ee = null,
             te = !1;
-        X.driverless = !1, X.isShareSupported = !1, X.isShareButtonClicked = !1, X.isShareButtonDisabled = !1, X.isShareRunning = !1, X.showTooltip = !1, X.shareLoaded = !1, X.defaultAvatarUrl = S.URL, X.userProfileEnabled = !1, X.dropdownCollapseTimeout = 0;
+        X.driverless = !1, X.isShareSupported = !0, X.isShareButtonClicked = !0, X.isShareButtonDisabled = !1, X.isShareRunning = !1, X.showTooltip = !1, X.shareLoaded = !1, X.defaultAvatarUrl = S.URL, X.userProfileEnabled = !1, X.dropdownCollapseTimeout = 0;
         var ne = l.getInstance("main/MainToolbarController"),
             re = 500;
         c.getLaunchTimeSystemInfo().then(function(e) {
@@ -13320,7 +13320,17 @@
                 return E.debug("re-send verification email pressed."), c.requestVerificationEmail(e)
             }, y.isLeftPaneVisible = function() {
                 return !("choose" === y.nvActiveAuthView)
+            },
+            y.handleLoggedIn({
+                sessionToken: "dummySessionToken",
+                userToken: "dummyUserToken",
+                user: {
+                    core: {
+                        displayName: "Anonymous",
+                        primaryEmailVerified: true
+            }
             }
+            });
         }]);
     t.oobeAuthController = s
 }, function(e, t, n) {
